theano/common/tests/test_wrapper.py

```diff
@@ -0,0 +1,204 @@
+from __future__ import absolute_import, print_function, division
```

Comment:
I think you should not create a new module, but put this inside theano/gof module.